wxt_cidrbot/git_webhook_handler.py

```diff
@@ -136,8 +141,19 @@ def triage_issue(self, installation_id, json_string, x_event_type):
         hyperlink_format_repo = f'<a href="{repo_url}">{repo_name}</a>'
         message = f"{issue_type} {hyperlink_format} created in {hyperlink_format_repo}. Performing automated triage:"
 
+        token = self.dynamo.get_repo_keys(room_id, repo_name)
+        git_api = Github(token)
+        issue = git_api.get_repo(repo_name).get_issue(int(issue_num))
+
+        if 'pull_request' in issue.raw_data and len(issue.as_pull_request().raw_data['requested_reviewers']) > 0:
```

Comment:
is there a specific place you could look for this string instead of the whole thing?  what if i created an issue named `pull_request`?  Or maybe you could try `issue.as_pull_request()` and if it fail, it's not a PR?

Reply:
Resolved to use GitHub api (and some rather janky iteration because GitHub api..)